targets/TARGET_STM/TARGET_STM32F0/STM32Cube_FW/CMakeLists.txt

```diff
@@ -1,77 +1,77 @@
-        STM32F0xx_HAL_Driver/stm32f0xx_hal_can.c
-        STM32F0xx_HAL_Driver/stm32f0xx_hal_cec.c
-        STM32F0xx_HAL_Driver/stm32f0xx_hal_comp.c
-        STM32F0xx_HAL_Driver/stm32f0xx_hal_cortex.c
-        STM32F0xx_HAL_Driver/stm32f0xx_hal_crc.c
-        STM32F0xx_HAL_Driver/stm32f0xx_hal_crc_ex.c
-        STM32F0xx_HAL_Driver/stm32f0xx_hal_dac.c
-        STM32F0xx_HAL_Driver/stm32f0xx_hal_dac_ex.c
-        STM32F0xx_HAL_Driver/stm32f0xx_hal_dma.c
-        STM32F0xx_HAL_Driver/stm32f0xx_hal_exti.c
-        STM32F0xx_HAL_Driver/stm32f0xx_hal_flash.c
-        STM32F0xx_HAL_Driver/stm32f0xx_hal_flash_ex.c
-        STM32F0xx_HAL_Driver/stm32f0xx_hal_gpio.c
-        STM32F0xx_HAL_Driver/stm32f0xx_hal_i2c.c
-        STM32F0xx_HAL_Driver/stm32f0xx_hal_i2c_ex.c
-        STM32F0xx_HAL_Driver/stm32f0xx_hal_i2s.c
-        STM32F0xx_HAL_Driver/stm32f0xx_hal_irda.c
-        STM32F0xx_HAL_Driver/stm32f0xx_hal_iwdg.c
-        STM32F0xx_HAL_Driver/stm32f0xx_hal_pcd.c
-        STM32F0xx_HAL_Driver/stm32f0xx_hal_pcd_ex.c
-        STM32F0xx_HAL_Driver/stm32f0xx_hal_pwr.c
-        STM32F0xx_HAL_Driver/stm32f0xx_hal_pwr_ex.c
-        STM32F0xx_HAL_Driver/stm32f0xx_hal_rcc.c
-        STM32F0xx_HAL_Driver/stm32f0xx_hal_rcc_ex.c
-        STM32F0xx_HAL_Driver/stm32f0xx_hal_rtc.c
-        STM32F0xx_HAL_Driver/stm32f0xx_hal_rtc_ex.c
-        STM32F0xx_HAL_Driver/stm32f0xx_hal_smartcard.c
-        STM32F0xx_HAL_Driver/stm32f0xx_hal_smartcard_ex.c
-        STM32F0xx_HAL_Driver/stm32f0xx_hal_smbus.c
-        STM32F0xx_HAL_Driver/stm32f0xx_hal_spi.c
-        STM32F0xx_HAL_Driver/stm32f0xx_hal_spi_ex.c
-        STM32F0xx_HAL_Driver/stm32f0xx_hal_tim.c
-        STM32F0xx_HAL_Driver/stm32f0xx_hal_tim_ex.c
-        STM32F0xx_HAL_Driver/stm32f0xx_hal_tsc.c
-        STM32F0xx_HAL_Driver/stm32f0xx_hal_uart.c
-        STM32F0xx_HAL_Driver/stm32f0xx_hal_uart_ex.c
-        STM32F0xx_HAL_Driver/stm32f0xx_hal_usart.c
-        STM32F0xx_HAL_Driver/stm32f0xx_hal_usart_ex.c
-        STM32F0xx_HAL_Driver/stm32f0xx_hal_wwdg.c
-        STM32F0xx_HAL_Driver/stm32f0xx_ll_adc.c
-        STM32F0xx_HAL_Driver/stm32f0xx_ll_comp.c
-        STM32F0xx_HAL_Driver/stm32f0xx_ll_crc.c
-        STM32F0xx_HAL_Driver/stm32f0xx_ll_crs.c
-        STM32F0xx_HAL_Driver/stm32f0xx_ll_dac.c
-        STM32F0xx_HAL_Driver/stm32f0xx_ll_dma.c
-        STM32F0xx_HAL_Driver/stm32f0xx_ll_exti.c
-        STM32F0xx_HAL_Driver/stm32f0xx_ll_gpio.c
-        STM32F0xx_HAL_Driver/stm32f0xx_ll_i2c.c
-        STM32F0xx_HAL_Driver/stm32f0xx_ll_pwr.c
-        STM32F0xx_HAL_Driver/stm32f0xx_ll_rcc.c
-        STM32F0xx_HAL_Driver/stm32f0xx_ll_rtc.c
-        STM32F0xx_HAL_Driver/stm32f0xx_ll_spi.c
-        STM32F0xx_HAL_Driver/stm32f0xx_ll_tim.c
-        STM32F0xx_HAL_Driver/stm32f0xx_ll_usart.c
-        STM32F0xx_HAL_Driver/stm32f0xx_ll_usb.c
-        STM32F0xx_HAL_Driver/stm32f0xx_ll_utils.c
-        system_stm32f0xx.c
+        stm32f0xx_HAL_Driver/Legacy/stm32f0xx_hal_can_legacy.c
```

Comment:
Real name is with upper case : STM32F0xx_HAL_Driver ?

Reply:
I believe it was a search and replace oversight.
Thanks for pointing that out.

Reply:
Correct, fixing now